McAfee/mcafee_web_gateway/_meta/fields.yml

```diff
@@ -0,0 +1,54 @@
+mcafee.webgateway.application.name:
+  name: mcafee.webgateway.application.name
+  description: "The name of the web application requested"
+  type: keyword
+
+mcafee.webgateway.application.reputation:
+  name: mcafee.webgateway.application.reputation
+  description: "The reputation string of the web application requested"
+  type: keyword
+
+mcafee.webgateway.url.reputation:
+  name: mcafee.webgateway.url.reputation
+  description: "The reputation string of an url"
+  type: keyword
+
+mcafee.webgateway.url.reputation_code:
+  name: mcafee.webgateway.url.reputation_code
+  description: "The reputation code of an url"
+  type: number
+
+mcafee.webgateway.categories:
+  name: mcafee.webgateway.categories
+  description: "A list of categories"
+  type: text
+
+mcafee.webgateway.viruses:
+  name: mcafee.webgateway.viruses
+  description: "A list of virus name"
+  type: text
+
+mcafee.webgateway.ruleset.name:
+  name: mcafee.webgateway.ruleset.name
+  description: "The name of the ruleset matching the request"
+  type: keyword
```

Comment:
Couldn’t we use [`rule.ruleset`](https://www.elastic.co/guide/en/ecs/current/ecs-rule.html#field-rule-ruleset) instead?

Reply:
I missed this ECS field